resources/lang/admin/en/word_filters.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * @author  MyBB Group
+ * @version 2.0.0
+ * @package mybb/core
+ * @license http://www.mybb.com/licenses/bsd3 BSD-3
+ */
+
+return [
+    'title'                       => 'Word Filters',
+    'add_button'                   => 'Add Word Filter',
+    'find_field'                  => 'Find',
+    'replace_field'           => 'Replacement',
```

Comment:
I wouldn't bother trying to line all the values up, it just gets annoying trying to maintain it later down the line 😉